src/Client/GameClient.cpp

```diff
@@ -154,6 +161,16 @@ bool GameClient::createGameObjects(GameObjectsInit gameObjectsInit) {
     return true;
 }
 
+void GameClient::loadSounds(std::map<std::string, std::string> soundPaths) {
+    for (std::pair<std::string, std::string> element : soundPaths){
+        musicManager->addPath(element.first, element.second, false);
+    }
+
+    musicManager->addPath(MUSIC, "Sound_Effects/Music/SuperMarioBrosSong.mp3", true);
```

Comment:
Please check if in linux .wav works as well as with mp3, in mac only wav works ok